docs/modules/ROOT/pages/how-to-guides/how-to-guide-landing-page.adoc

```diff
@@ -0,0 +1,29 @@
+== How-to guides
+Guides to help you complete a specific goal.
+
+xref:how-to-guides/Secure-your-supply-chain/proc_upgrade_build_pipeline.adoc[**Upgrading the build pipeline**]
+Provides procedures for upgrading the build pipeline.
+
+xref:how-to-guides/Secure-your-supply-chain/proc_inspect_sbom.adoc[**Inspecting Software bill of materials (SBOMS)**]
+Provides procedures on downloading SBOMS through the UI and using the CLI.
+
+xref:how-to-guides/Secure-your-supply-chain/proc_java_dependencies.adoc[**Configuring dependencies rebuild for Java applications**]
+Provides procedures on configuring dependencies rebuild for Java applications
+
+
+xref:how-to-guides/proc_creating_your_own_environment.adoc[**Creating your own environment**]
+Provides procedures on creating your own environment.
+
+xref:how-to-guides/creating_a_custom_application_test_with_test_pipelines.adoc[**Creating a Custom Application Test with Test Pipelines**]
+Provides procedures on how integration services use Tekton test pipelines and test scenarios.
+
+link:placeholder.com[Testing applications]
```

Comment:
I see there's a "placholder" link and I deduce that's why I get a 404 error when I click the link. 

Should we even include these guides if we don't have content for them yet?